source/modules/MEI.shared.xml

```diff
@@ -2764,6 +2764,18 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.rotating" module="MEI.shared" type="atts">
+    <desc>Visual domain attributes.</desc>
```

Comment:
This description should probably be updated -- "Attributes that describe rotation"?